package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@ainft-team/ainft-js",
   "main": "dist/ainft.js",
   "types": "dist/ainft.d.ts",
-  "version": "0.5.14",
+  "version": "0.5.15",
```

Comment:
question) when the minor version changes?

Reply:
we will update minor version when merged chain/network separation pr.
and major version maybe will update with authentication updates.

Reply:
sounds good, thanks!